tests/modules/ecs6-class/line.test.js

```diff
@@ -0,0 +1,138 @@
+const Line = require("../../../modules/ecs6-class/line");
+const Point = require("../../../modules/ecs6-class/point");
+
+
+describe("LINE", function () {
+    it("should return point1 of line", function () {
+        const x = new Line({ point1: new Point({ x: 2, y: 3 }), point2: new Point({ x: 2, y: 3 }) })
+        expect(x.point1).toStrictEqual(new Point({ x: 2, y: 3 }));
+    })
+    describe("CALCULATE_SLOPE", function () {
+        it("should return slope of line", function () {
+            const poix = new Point({ x: 3, y: 4 });
+            const poiy = new Point({ x: 1, y: 3 });
+            const li1 = new Line({ point1: poix, point2: poiy })
+            li1.calculateSlope()
+            expect(li1.slope).toBe(0.5);
+        })
+    })
+
+    describe("CALCULATE_N_OF_LINE_FUNCTION", function () {
+        it("should return n of line", function () {
+            const poix = new Point({ x: 3, y: 4 });
+            const poiy = new Point({ x: 1, y: 3 });
+            const li1 = new Line({ point1: poix, point2: poiy })
+            li1.calculateSlope()
+            li1.calculateNOfLineFunction()
```

Comment:
if you are tests calculateNOfLineFunction - don't execute different functions in the tests